src/ServiceControl/Licensing/LicenseController.cs

```diff
@@ -27,7 +27,8 @@ public ActionResult<LicenseInfo> License(bool refresh)
                 Status = activeLicense.IsValid ? "valid" : "invalid",
                 LicenseType = activeLicense.Details.LicenseType ?? string.Empty,
                 InstanceName = settings.InstanceName ?? string.Empty,
-                LicenseStatus = activeLicense.Details.Status
+                LicenseStatus = activeLicense.Details.Status,
+                ConnectedApplications = settings.ConnectedApplications.ToArray()
```

Comment:
Should we make it a persistent document with an in-memory cache to avoid writes if we already have a specific value there?

Reply:
Connected applications are now persisted, and they've been moved off of the license object to the root controller